internal/controllers/user_devices_controller_test.go

```diff
@@ -400,8 +404,13 @@ func (s *UserDevicesControllerTestSuite) TestGetMyUserDevices() {
 	)
 	_ = test.SetupCreateAutoPiUnit(s.T(), testUserID, unitID, func(s string) *string { return &s }(deviceID), s.pdb)
 	_ = test.SetupCreateUserDeviceAPIIntegration(s.T(), unitID, deviceID, ud.ID, integration.Id, s.pdb)
+
+	addr := "67B94473D81D0cd00849D563C94d0432Ac988B49"
```

Comment:
I think the goal of these test additions is a bit unclear. Are we just trying to prevent an NPE?